src/libs/Visibility/index.desktop.ts

```diff
@@ -4,28 +4,20 @@ import ELECTRON_EVENTS from '../../../desktop/ELECTRON_EVENTS';
  * Detects whether the app is visible or not. Electron supports document.visibilityState,
  * but switching to another app while Electron is partially occluded will not trigger a state of hidden
  * so we ask the main process synchronously whether the BrowserWindow.isFocused()
- *
- * @returns {Boolean}
  */
 function isVisible() {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
     return window.electron.sendSync(ELECTRON_EVENTS.REQUEST_VISIBILITY);
 }
 
-/**
- * @returns {Boolean}
- */
 function hasFocus() {
```

Comment:
Please create `types.ts` file with shared types, similar to this [PR](https://github.com/Expensify/App/pull/27022/files)